gateway/tracing.go

```diff
@@ -42,6 +43,7 @@ func (tr *traceHttpRequest) toRequest(ignoreCanonicalMIMEHeaderKey bool) (*http.
 type traceRequest struct {
 	Request *traceHttpRequest     `json:"request"`
 	Spec    *apidef.APIDefinition `json:"spec"`
+	OAS     *oas.OAS              `json:"oas"`
```

Comment:
```suggestion
	model.MergedAPI
```

I'm sort of worried these have `json` keys in the object, why?

Reply:
I think dismiss